fhir-server/src/test/java/au/csiro/pathling/test/builders/UntypedResourcePathBuilder.java

```diff
@@ -84,6 +88,12 @@ public UntypedResourcePathBuilder singular(final boolean singular) {
     return this;
   }
 
+  @Nonnull
+  public UntypedResourcePathBuilder thisColumn(final Column thisColumn) {
```

Comment:
Is the nullability annotation missing here or do we have a conversion that if not annotated == @Nullable?

Reply:
See 0c7f8ee500c313ee0ffa716fda34c44c6f68686a.